README.md

```diff
@@ -0,0 +1,58 @@
+# AWESOME BOOKS
+![](https://github.com/brytebee/AwesomeBooks)
+
+## First JS Project: Awesome Project
+
+> In this project, you will build a basic website that allows users to add/remove books from a library list of books. You will achieve that by using JavaScript objects and arrays. You will also need to dynamically modify the DOM and add basic events.
+
+![image](https://user-images.githubusercontent.com/43698511/128846370-dc5622fe-b062-4655-afa8-97b2a53202af.png)
+
+This project features HTML5 semantic tags, CSS and JS-DOM.
+
+## Built With
+
+- HTML5, CSS3 and JS-DOM
+
+## Live Demo
+
+[Live Demo Link](https://brytebee.github.io/)
```

Comment:
- [ ] OPTIONAL 
This demo link yields a _404 error_, thus page cannot be found. Kindly update this link with the correct GitHub pages host link.